docs/marketplace/fastlane/testing-distribution/index.md

```diff
@@ -58,6 +61,8 @@ fastlane add_plugin appcircle_testing_distribution
 - `personalAPIToken`: The Appcircle Personal API token is used to authenticate and secure access to Appcircle services. Add this token to your credentials to enable its use in your pipeline and ensure authorized actions within the platform.
 - `profileName`: Specifies the profile that will be used for uploading the app.
 - `createProfileIfNotExists`: Ensures that a user profile is automatically created if it does not already exist; if the profile name already exists, the app will be uploaded to that existing profile instead.
+- `profileCreationSettings`: If `createProfileIfNotExists` is `true` and a new profile being created, the profile will be configured with this settings.
```

Comment:
Since this parameter depends on an **optional** parameter, it should also be optional.

We should add " (optional)" as it was done in the PR below.
- https://github.com/appcircleio/appcircle-docusaurus/pull/763/files